sphinx/texinputs/sphinxlatexobjects.sty

```diff
@@ -77,56 +77,79 @@
 
 % Signatures, possibly multi-line
 %
+% For legacy reasons Sphinx uses LaTeX \list and \item's for signatures
+% This is delicate:
+% - the actual item label is not typeset immediately by \item but later as part
+% of the \everypar which will be triggered by either next paragraph or a manual
+% \leavevmode, or if nothing in-between by the next \item,
+% - \begingroup <set-up>\item[foo] <setup>\endgroup leads to errors,
+% - vertical space depends on \parskip and \itemsep values in somewhat
+% subtle manners.
+%
+% Since the 2022/01/13 version things are simpler as \parskip is simply set
+% to zero during execution of \pysigline/\pysiglinewithargsret
+%
+% Parameter for separation via \itemsep of multiple signatures with common desc
+\newlength\sphinxsignaturesep
+\setlength\sphinxsignaturesep{\smallskipamount}
```

Comment:
I experimented with 1pt, 2pt, 3pt with lots of hesitation. FInally `\smallskipamount` (which is `3pt` with some stretch/shrink in default LaTeX) as perhaps it is better, e.g. for Japanese?

Reply:
I really like the generated output, a nice improvement! Thanks for it.